AFQ/api.py

```diff
@@ -182,14 +188,48 @@ def __init__(self,
         b0_threshold : int, optional
             The value of b under which it is considered to be b0. Default: 0.
 
+        target_b_val : float, optional
+            Which b value you want to use from the dataset.
+            If None, all are used. Default: None
+
+        target_b_sensitivity : float, optional
+            Sensitivity of target_b_val.
+            Only used if target_b_val is not None.
+            B values selected will be in the range
+            target_b_val +/- target_b_sensitivity
+            Default: 50
+
         odf_model : string, optional
-            Which model to use for determining directions in tractography
-            {"DTI", "DKI", "CSD"}. Default: "DTI"
+            Which model to use for determining directions in tractography.
+            Can be one of: {"DTI", "DKI", "CSD"}. Default: "DTI"
 
         directions : string, optional
             How to select directions for tracking (deterministic or
             probablistic) {"det", "prob"}. Default: "det".
 
+        reg_subject : str or Nifti1Image, optional
+            The source image data to be registered.
+            Can either be a Nifti1Image, a path to a Nifti1Image, or
+            If "mni_T2", "b0", "dti_fa_subject", "dti_fa_template",
+            "subject_sls", "hcp_atlas", "mni_T1", or "power_map,"
+            image data will be loaded automatically.
+
+        reg_template : str or Nifti1Image, optional
+            The target image data for registration.
+            Can either be a Nifti1Image, a path to a Nifti1Image, or
+            If "mni_T2", "b0", "dti_fa_subject", "dti_fa_template",
+            "subject_sls", "hcp_atlas", "mni_T1", or "power_map,"
+            image data will be loaded automatically.
+            (defaults to the MNI T2)
```

Comment:
I am a little bit confused here: `reg_subject` is in the subject native space and `reg_template` is in some (usually MNI for now) template space, right? 

So it doesn't make sense to use `reg_subject="mni_T2"` or `reg_subject="mni_T1"` or, conversely `reg_template="dti_fa_subject"` or `reg_template="power_map"`. Or am I missing something?

 This can never be `dti_fa_subject` or `power_map`, right? We only have these for the individual subject right now.

Reply:
I will update the documentation so that only sensible options are listed here.